pennylane_qiskit/converter.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+QuanctumCircuit converter module
+================================
+
+.. currentmodule:: pennylane_qiskit.load
+
+This module contains functions for converting Qiskit QuantumCirducit objects
+PennyLane templates.
+"""
+import warnings
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.exceptions import QiskitError
+
+import pennylane.ops.qubit as pennylane_ops
+from pennylane_qiskit.qiskit_device import QISKIT_OPERATION_MAP
+
+# pylint: disable=too-many-instance-attributes
+
+inv_map = {v.__name__: k for k, v in QISKIT_OPERATION_MAP.items()}
+
+
+class WiresError(Exception):
+    """Exception raised by  the pennylane_qiskit.converter when it encounters an illegal
+    operation in the quantum circuit.
+    """
```

Comment:
The intention here was to raise a custom error specifically related to the number of wires. I do agree though, that ``qml.QuantumFunctionError`` can also be raised with a message describing the situation. It might be even better, because then each plugin can just raise this type of error, and they do not have to define a custom error type.

Modifying these, thanks!